src/NBitcoin/QBitNinjaTransactionRepository.cs

```diff
@@ -6,61 +6,50 @@ namespace NBitcoin
 {
     public class QBitNinjaTransactionRepository : ITransactionRepository
     {
-        private readonly Uri _BaseUri;
-        public Uri BaseUri
```

Comment:
Are you sure the public `get` for BaseUri is not used anywhere? (might not be, just asking)

Reply:
Hmmm you are right, it be used in other projects, I'll make this public again, just in case.